test/fuse_reduce.cpp

```diff
@@ -133,6 +134,53 @@ TEST_CASE(pointwise_reduce)
     EXPECT(p1 == p2);
 }
 
+TEST_CASE(scalar_multibroadcast)
+{
+    // Matches the find_pointwise_reduce matcher, but input x has a (scalar) shape
+    // incompatible with the multibroadcast instruction; therefore it
+    // creates a fused_reduce module but does not add a submodule for the
+    // multibroadcast instruction. 
+    migraphx::shape sdot{migraphx::shape::half_type, {80, 204, 204}};
+    migraphx::shape sdot_double{migraphx::shape::double_type, {80, 204, 204}};
+    migraphx::shape scalar{migraphx::shape::double_type, {1}, {0}};
```

Comment:
Just use the same data type for all shapes for the test to make it simpler.

Reply:
done.